test/unit/app/models/edition/custom_lead_image_test.rb

```diff
@@ -35,14 +35,15 @@ def body_text_valid(body)
     assert body_text_valid("foo\[Image: non_lead_image.jpg]\nbar")
   end
 
-  test "#update_lead_image updates the lead_image association to the oldest image" do
-    image1 = build(:image)
+  test "#update_lead_image updates the lead_image association to the oldest non-svg image" do
+    svg_image_data = build(:image_data, file: File.open(Rails.root.join("test/fixtures/images/test-svg.svg")))
+    image1 = build(:image, image_data: svg_image_data)
```

Comment:
I know the original test didn't do it, but do you think it might be worth setting the created timestamp on these images just to be really explicit about how this should work? I don't like relying on the implicit order too much.